src/btree/bt_vrfy.c

```diff
@@ -67,10 +75,20 @@ __verify_config(WT_SESSION_IMPL *session, const char *cfg[], WT_VSTUFF *vs)
     WT_RET(__wt_config_gets(session, cfg, "dump_pages", &cval));
     vs->dump_pages = cval.val != 0;
 
+    WT_RET(__wt_config_gets(session, cfg, "stable_timestamp", &cval));
+    vs->stable_timestamp = WT_TS_NONE; /* Ignored unless a value has been set */
+    if (cval.val != 0) {
+        if (!txn_global->has_stable_timestamp)
+            WT_RET_MSG(
+              session, ENOTSUP, "the database does not have a stable timestamp to verify against");
```

Comment:
That is awkward wording. I suggest something like: "cannot verify without a stable timestamp set"